docs/troubleshooting.md

```diff
@@ -0,0 +1,21 @@
+# Troubleshooting Guide
+
+## Other troubleshooting guides
```

Comment:
💯

Reply:
Wonder if it might be useful to highlight what kinds of issues the docs here solve.  As an example noting generally what kinds of items might be in the mux doc vs the one here.